core/quantity/README.md

```diff
@@ -1,11 +1,230 @@
 # @itwin/core-quantity
 
-Copyright © Bentley Systems, Incorporated. All rights reserved. See LICENSE.md for license terms and full copyright notice.
-
-## Description
+- [@itwin/core-quantity](#itwincore-quantity)
+  - [Terms and Concepts](#terms-and-concepts)
+    - [Common Terms](#common-terms)
+    - [Concepts](#concepts)
+      - [Units Provider](#units-provider)
+      - [Unit Conversion](#unit-conversion)
+    - [Examples of Usage](#examples-of-usage)
+      - [Numeric Format](#numeric-format)
+      - [Composite Format](#composite-format)
+      - [Parsing Values](#parsing-values)
+      - [Mathematical Operation Parsing](#mathematical-operation-parsing)
+  - [Contributing](#contributing)
+- [Licensing](#licensing)
 
 The __@itwin/core-quantity__ package contains classes for quantity formatting and parsing.
 
-## Documentation
+## Terms and Concepts
+
+### Common Terms
+
+- [Unit]($quantity)/[UnitProps]($quantity) - A named unit of measure which can be located by its name or label. The definition of any unit is represented through it's `UnitProps`.
```

Comment:
Instead of using $quantity here and everywhere else in this file, should I link straight to the line of code of the relevant interfaces and methods? Assuming this readme isn't part of the docs site

Reply:
I'm not familiar with using variables in markdown like this. Where do we define $quantity? The github preview for the file turns this into dead links.
The idea of putting line of code links here does not appeal to me. I've never seen that being done in abstract code documentation. People will read this .md to understand concepts. Also code links could easily go out of sync.

Reply:
As an example, see this [README.md](https://github.com/iTwin/itwinjs-core/blob/master/docs/learning/frontend/QuantityFormatting.md?plain=1), and this [link](https://www.itwinjs.org/learning/frontend/quantityformatting/) shows how it is displayed in the docs site. So, these $<XXX_packagename> links really only make sense in the context of our itwinjs.org docs site. So I think I will remove the $quantity and everywhere else... I agree with the code links, I won't introduce them. Instead, I'll put a link at the top of the opening description, linking to the docs site's API reference for core-quantity

Reply:
okay, added